site/lib/src/components/client/archive_table.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright 2025 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:jaspr/jaspr.dart';
+import 'package:meta/meta.dart';
+import 'package:universal_web/js_interop.dart';
+
+class Release {
```

Comment:
Consider moving this to `/site/lib/src/models/release_model.dart` since its shared between both components and renaming the class something like `FlutterRelease` to more clearly signify what it represents.